src/ui/MLFacet/MLFacet.ts

```diff
@@ -167,11 +167,7 @@ export class MLFacet extends Component {
 
     const facetResponse = findWhere(data.results.facets, { field: this.fieldName });
 
-    if (!facetResponse) {
-      this.fieldDoesNotExistError();
-    }
-
-    this.onQueryResponse(facetResponse.values);
+    this.onQueryResponse(facetResponse ? facetResponse.values : []);
```

Comment:
Under what condition(s) would we not get a response?

Reply:
Facet is not relevant anymore
Facet has no values and selection
Not having the facet in the response would hide it in the UI